### PR TITLE
Add example program to dropdown

### LINKED ISSUE #84: Add the "example Program" to the dropdown.
At the moment, the example program that is shown when the playground starts cannot be chosen by the user using the dropdown. We can add this example to the dropdown.

### PR DESCRIPTION
**Pull request by Dominik1999**
Closes #84 

And I added the pre-commit yaml file

### CODE REVIEW

File: playground/src/pages/CodingEnvironment.tsx [hunk 1]
```diff
@@ -198,7 +203,7 @@ export default function CodingEnvironment(): JSX.Element {
         setOutput(formatDebuggerOutput(debugOutput));
       }
     } catch (error) {
-      setOutput("Error: Check the developer console for details.");
+      setOutput("Error: Check the developer AA console for details.");
```
Comment (hackaugusto) [inline playground/src/pages/CodingEnvironment.tsx:206]:
```suggestion
      setOutput("Error: Check the developer console for details.");
```

File: playground/src/utils/helper_functions.tsx [hunk 2]
```diff
@@ -177,7 +177,6 @@ export function formatMemory(memory: BigUint64Array): string {
   let output = "";
   for (let i = 0; i < memory.length; i += 5) {
     const memorySlice = memory.slice(i, i + 5);
-    output += `[${memorySlice[0]}]: [${memorySlice[1]}, ${memorySlice[2]}, ${memorySlice[3]}, ${memorySlice[4]}] \n                    `;
+    output += `[${memorySlice[0]}]: [${memorySlice[1]}, ${memorySlice[2]}, ${memorySlice[3]}, ${memorySlice[4]}] \n           `;
```
Comment (hackaugusto) [inline playground/src/utils/helper_functions.tsx:180]: what is the rest of the space for?

File: playground/src/utils/helper_functions.tsx [hunk 3]
```diff
@@ -177,7 +177,6 @@ export function formatMemory(memory: BigUint64Array): string {
...
   }
-  return output;
-}
+  return output;}
```
Comment (hackaugusto) [inline playground/src/utils/helper_functions.tsx:182]: this seems unecessary

File: playground/__tests__/Components.test.tsx [hunk 4]
```diff
@@ -24,16 +25,16 @@ describe("ActionButton component", () => {
...
 
-    fireEvent.mouseDown(getByRole("button"));
-    const listbox = within(getByRole("listbox"));
+    fireEvent.mouseDown(getByRole('button'));
+    const listbox = within(getByRole('listbox'));
 
-    fireEvent.click(listbox.getByText("comparison"));
-    expect(getByTestId("listbox")).toHaveTextContent("comparison");
+    fireEvent.click(listbox.getByText("example"));
+    expect(getByTestId('listbox')).toHaveTextContent('example');
```
Comment (hackaugusto) [inline playground/__tests__/Components.test.tsx:34]: the quotes are inconsistent ... please check that you're running your formatter properly

here is how to setup prettier: https://prettier.io/docs/en/precommit.html

**Review by hackaugusto — CHANGES_REQUESTED** [review]

File: playground/src/utils/helper_functions.tsx (hunk 2 above)
Comment (Dominik1999) [reply playground/src/utils/helper_functions.tsx:180]: It's for alignment of the different memory slots

File: playground/__tests__/Components.test.tsx (hunk 4 above)
Comment (Dominik1999) [reply playground/__tests__/Components.test.tsx:34]: done, will set up

**Comment by Dominik1999** [comment]
@hackaugusto I understand that this is messier than it should be. My `pre-commit` is not set up properly, so sometimes it formats and sometimes it doesn't. 

I will set this up after zkHack. Today I don't have enough time. Maybe we can merge this as is now

**Review by hackaugusto — APPROVED** [review]

**Pull request merged**